src/org/javarosa/core/io/Std.java

```diff
@@ -0,0 +1,32 @@
+package org.javarosa.core.io;
+
+import java.io.PrintStream;
+
+/**
+ * Used to break dependency directly on System.out and System.err.
+ */
+public class Std {
+
+    public static PrintStream out = System.out;
```

Comment:
Would it be better to always force use of the setters by making the class vars private?